network/src/connectivity_manager/mod.rs

```diff
@@ -120,7 +120,7 @@ pub struct ConnectivityManager<TBackoff> {
 #[repr(u8)]
 #[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, NumVariants, Serialize)]
 pub enum DiscoverySource {
-    OnChain,
+    ValidatorSet,
```

Comment:
not sure I agree with that, since a ValidatorSet could be a configuration file?

Reply:
We could do `OnChainValidatorSet`, but I want it to be very clear with the new onchain stuff after this, for which one is which as to not collide & confuse

Reply:
ahh, good callout